config/sources/families/sun50iw9.conf

```diff
@@ -15,47 +15,65 @@ GOVERNOR=ondemand
 case $BRANCH in
 
 	legacy)
```

Comment:
There has been no change in `legacy` of `orangepi zero2` here, only the addition of `bigtreetech cb1` configuration under `legacy`